src/test/java/org/folio/rest/api/InstanceBulkTest.java

```diff
@@ -0,0 +1,400 @@
+package org.folio.rest.api;
+
+import io.vertx.core.Vertx;
+import io.vertx.core.buffer.Buffer;
+import io.vertx.core.http.HttpClientRequest;
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+import io.vertx.ext.unit.Async;
+import io.vertx.ext.unit.TestContext;
+import io.vertx.ext.unit.junit.VertxUnitRunner;
+
+import org.folio.rest.jaxrs.model.InstanceBulkIdsGetField;
+import org.folio.rest.persist.PostgresClient;
+import org.folio.rest.support.*;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.UUID;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeoutException;
+
+import static java.net.HttpURLConnection.HTTP_OK;
+import static java.util.concurrent.TimeUnit.SECONDS;
+import static org.folio.rest.api.StorageTestSuite.TENANT_ID;
+import static org.folio.rest.support.ResponseHandler.json;
+import static org.folio.rest.support.http.InterfaceUrls.*;
+import static org.folio.util.StringUtil.urlEncode;
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+@RunWith(VertxUnitRunner.class)
+public class InstanceBulkTest extends TestBaseWithInventoryUtil {
+  private static final Logger log =
+    LoggerFactory.getLogger(InstanceBulkTest.class);
+
+  private Set<String> instanceIdsToRemoveAfterTest = new HashSet<>();
+
+  @Before
+  public void beforeEach() {
+    StorageTestSuite.deleteAll(itemsStorageUrl(""));
+    StorageTestSuite.deleteAll(holdingsStorageUrl(""));
+    StorageTestSuite.deleteAll(instancesStorageUrl(""));
+  }
+
+  @After
+  public void resetInstanceHRID() {
+    setInstanceSequence(1);
+  }
+
+  @After
+  public void checkIdsAfterEach() {
+    StorageTestSuite.checkForMismatchedIDs("instance");
+  }
+
+  @After
+  public void removeGeneratedEntities(TestContext context) {
+    final Async async = context.async();
+    List<CompletableFuture<Response>> cfs =
+      new ArrayList<CompletableFuture<Response>>();
+
+    instanceIdsToRemoveAfterTest.forEach(id -> cfs.add(client
+      .delete(instancesStorageUrl("/" + id), TENANT_ID)));
+    CompletableFuture.allOf(cfs.toArray(new CompletableFuture[cfs.size()]))
+      .thenAccept(v -> async.complete());
+  }
```

Comment:
The `TestBaseWithInventoryUtil` should remove the entire table before each test, so we can remove this method.